taxi_service/settings.py

```diff
@@ -57,7 +57,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [BASE_DIR / "templates"],
+        "DIRS": [os.path.join(BASE_DIR, "templates")],
```

Comment:
it's not necessary cause / overloads work with str to path joining
but ok, let it be

Reply:
Ok, thanks for info